docs-rst/installation/server-config/security.rst

```diff
@@ -29,27 +30,46 @@ You can use KSQL with a Kafka cluster in |ccloud|. For more information, see :re
 Configuring KSQL for Secured Confluent Schema Registry
 ------------------------------------------------------
 
-The following configuration connects KSQL with the Confluent Schema Registry over HTTPS.
+KSQL can be configured to connect to the Schema Registry over HTTP by setting the
+``ksql.schema.registry.url`` to the Schema Registry's HTTPS endpoint.
+Depending on your security setup, you may also need to supply a trust-store, (e.g. if the Schema
+Registry's SSL certificates are not trusted they the JVM by default), and a key-store, (e.g. if the
+Schema Registry requires mutual authentication).
 
-#. Specify the HTTPS endpoint in the ``ksql.schema.registry.url`` setting in the
-   KSQL server configuration file:
+The following configuration is all that is needed to configure KSQL to communicate with the Schema Registry
+over HTTPS, where mutual authentication is not required and the Schema Registry's SSL certificates
+are trusted by the JVM:
 
-   .. code:: bash
+.. code:: bash
+
+    ksql.schema.registry.url=https://<host-name-of-schema-registry>:<ssl-port>
+
+The following configuration can be used to configure KSQL to communicate with the Schema Registry over HTTPS,
```

Comment:
Simplify as: `To configure KSQL to communicate with the Schema Registry over HTTPS, with mutual authentication (i.e. SSL configuration is shared between Kafka and Schema Registry):`

Reply:
This reworking doesn't work: 'mutual authentication' is not the same thing as sharing config between Kafka and Schema Registry.  

Mutual authentication is a for of SSL communication where both parties authenticate with each other using SSL certificates. 

Shared configuration relates to the fact that you can configure SSL in the SR client using either prefixed or non prefixed configuration names.  e.g. you can use either `ssl.truststore.location` or `ksql.schema.registry.ssl.truststore.location`.  The former is used where the same SSL configuration is needed for communicating with both Kafka and the SR. The latter overrides the non-prefixed and only configures SR SSL comms.

I've added a part to explain this at the top of the section and reworked this paragraph. See what you think.